### PR TITLE
[WOR-464] Set up dependabot

### PR DESCRIPTION
**Pull request by yunip20**
https://broadworkbench.atlassian.net/browse/WOR-464

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+version: 2
+updates:
+  # Enable version updates for Gradle
+  - package-ecosystem: "gradle"
+    directory: "/"
+    schedule:
+      interval: "daily"
```
Comment (cahrens) [inline .github/dependabot.yml:7]: We may not want to do this daily. What are the options?

Comment (yunip20) [reply .github/dependabot.yml:7]: daily—runs on every weekday, Monday to Friday.
weekly—runs once each week. By default, this is on Monday. (can modify this)
monthly—runs once each month. This is on the first day of the month.

**Review by cahrens — APPROVED** [review]
Let's get this in and iterate as needed.

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    target-branch: "main"
+    reviewers:
+      - "@DataBiosphere/broadworkspaces"
+      - "@rtitle"
+    labels:
+      - "dependency"
+      - "gradle"
+    commit-message:
+      prefix: "[No Ticket]"
```
Comment (cahrens) [inline .github/dependabot.yml:16]: Yunyi is following examples from other repos-- for instance, https://github.com/DataBiosphere/duos-ui/blob/develop/.github/dependabot.yml.

File: .github/dependabot.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+version: 2
+updates:
+  # Enable version updates for Gradle
+  - package-ecosystem: "gradle"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    target-branch: "main"
```
Comment (msilva-broad) [inline .github/dependabot.yml:8]: Is the default for dependabot to NOT auto-merge PRs with the dependency updates?
Also, will this setup kick off the PR pipeline checks?  Specifically Unit Tests run?

File: .github/dependabot.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    reviewers:
+      - "@DataBiosphere/broadworkspaces"
+      - "@rtitle"
```
Comment (aherbst-broad) [inline .github/dependabot.yml:11]: Not sure we want to bug Rob with these PRs, I think the `broadworkspaces` team should be ok.

**Review by aherbst-broad — APPROVED** [review]

File: .github/dependabot.yml (hunk 3 above)
Comment (yunip20) [reply .github/dependabot.yml:8]: The default for dependabot does not auto-merge PRs with updates. This is an example of a PR that dependabot creates: https://github.com/DataBiosphere/duos-ui/pull/1740

**Pull request merged**